_posts/newsletters/2018-05-25-pydata-amsterdam-2018.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Newsletter 2020-04-21
+category: [newsletter]
+---
+
+Due to the unfortunate circumstances surrounding COVID-19, we can not come together in person. That doesn't mean though that PyData Amsterdam is done. Over the years we've been building up a fantastic repertoire of recordings at both PyData Amsterdam and at other PyData events across the world. In the coming period, we'll highlight some of our favourites for you to enjoy as well as keep you updated with some of our and the rest of the communities’ plans.
+
+## Our picks:
+
+### The evolution of array computing in Python - Ralf Gommers (PyData Amsterdam 2019)
+https://www.youtube.com/watch?v=HVLPJnvInzM
+
+Have you ever wanted to run your NumPy based code on multiple cores, or on a distributed system, or on your GPU? Wouldn't it be nice to do this without changing your code? In this talk, Ralf describes the addition of two protocols to numpy which attempt to provide a bridge between those different array implementations
+
+Ralf has been deeply involved in the SciPy and PyData communities for over a decade. He is a core developer of NumPy, SciPy and PyWavelets, and has contributed widely throughout the SciPy ecosystem.
+
+
+### Let them write code - Ines Montani (PyCon India 2019)
+https://www.youtube.com/watch?v=Ivb4AAuj5JY
```

Comment:
Shall we make all youtube links into embeds? It seems to work nicely even though Markdown doesn't officially support it.

```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/Ivb4AAuj5JY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```